docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,35 @@
+[[type-definitions-schema-configuration]]
+= Schema Configuration
+
+Neo4j GraphQL Library supports several functionalities such as CRUD operations, aggregation, filtering, and others.
+To make them work, a large amount of GraphQL types are generated.
+
+For instance, starting from the type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String
+    length: Int
+} 
+----
+
+In some cases, it may be advisable to reduce the scope of the API produced. Whatever the reason is to reduce the size of the schema, improve the performance, or limit access to unwanted operations, this section provides all the information needed to configure the API produced.
+
+From the type definitions above, let's say we want to prevent users from querying `movies` in the Neo4j Database. To do that, let's bring some changes to the initial type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie @query(read: false, aggregate: false) { 
+    title: String
+    length: Int
+} 
+----
+
+With this change, the library does **not** generate the following operation fields from the `Query` type:
+
+* `movies`
+* `moviesAggregate`
+* `moviesConnection`
+
+NOTE: Please note that aggregations will no longer be generated by default in the 4.x version of the library. See xref::guides/v4-migration/index.adoc#opt-in-aggregation[`Opt-in Aggregation`]
```

Comment:
In other places you are using 4.0.0 instead of 4.x when speaking about the version. Is there a more correct or preferred way to refer to the version?

```suggestion
[NOTE]
====
Aggregations will no longer be generated by default in the 4.x version of the library. See xref::guides/v4-migration/index.adoc#opt-in-aggregation[`Opt-in Aggregation`] for more information.
====
```

Reply:
I would say 4.0.0!